monai/data/meta_tensor.py

```diff
@@ -113,8 +114,8 @@ def __torch_function__(cls, func, types, args=(), kwargs=None) -> torch.Tensor:
         ret.affine = ret.affine.to(ret.device)
         return ret
 
-    def get_default_affine(self) -> torch.Tensor:
-        return torch.eye(4, device=self.device)
+    def get_default_affine(self, dtype=torch.float64) -> torch.Tensor:
```

Comment:
64 or 32? I feel like the default is normally 32?

Reply:
normally 64 is better when resampling the continuous coordinates...